modules/cnf-topology-aware-lifecycle-manager-cluster-logging-6x-update-policygenerator.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="talm-cluster-logging-6x-update-policygenerator_{context}"]
+= Removing Cluster Logging Operator artifacts with {policy-gen-cr} CRs
+
+When you update to Cluster Logging Operator version 6.x from version 5.x, you must add a cleanup policy that removes the old Operator artifacts from deployed clusters after creating the new API custom resources (CRs).
+
+.Procedure
+
+. Add the `acm-group-du-clo5-cleanup.yaml` file in the git repository that contains your other `PolicyGenerator` CRs.
+
++
+.Example cleanup files
+[source,yaml]
+----
+apiVersion: policy.open-cluster-management.io/v1
+kind: PolicyGenerator
+metadata:
+  name: group-du-sno-clo5-cleanup
+placementBindingDefaults:
+  name: group-du-sno-clo5-cleanup-placement-binding
+policyDefaults:
+  namespace: ztp-group
+  # Allow this policy to be unbound from clusters based on a custom label
+  placement:
+    labelSelector:
+      matchExpressions:
+      - key: group-du-sno
+        operator: Exists
+      - key: du-profile
+        operator: In
+        values: ["latest"]
+      - key: clo5-cleanup-done
+        operator: DoesNotExist
+  remediationAction: inform
+  severity: low
+  # standards: []
+  namespaceSelector:
+    exclude:
+      - kube-*
+    include:
+      - '*'
+  evaluationInterval:
+    compliant: 10m
+    noncompliant: 10s
+policies:
+- name: group-du-sno-clo5-cleanup
+  policyAnnotations:
+    ran.openshift.io/ztp-deploy-wave: "11" <1>
+  manifests:
+    - path: source-crs/ClusterLogging5Cleanup.yaml
+----
+<1> Default value is `11`. Ensure that the wave of `group-du-sno-clo5-cleanup` policy is after the `group-du-sno` configuration policy, which is wave `10` by default.
+
+. Update the binding rules in the cleanup `PolicyGenerator` CR as needed to match the labels in use for the associated fleet of clusters.
+
+. Add the cleanup `PolicyGenerator` CR to the `generators` section in the `kustomization.yaml` file.
+
++
+.Example `kustomization.yaml` file
+[source,yaml]
+----
+generators:
+- acm-common-ranGen.yaml
+# ...
+- acm-group-du-clo5-cleanup.yaml
+# ...
+----
+
+. Commit the cleanup `PolicyGenerator` CR and `kustomization.yaml` file, and then push to their git repository.
+
+. Wait for ArgoCD to generate and sync the cleanup policy to the hub cluster.
+
+.Verification
+
+. Verify that the cleanup policy is bound to all clusters that upgraded from Cluster Logging Operator version 5.x to 6.x.
+
+. Verify that the cleanup policy wave is after the `group-du-sno` configuration policy.
```

Comment:
Will folks know how to do these steps?

Reply:
@mburke5678 The customers utilizing TALM will be familiar with addressing these steps.